Documentation/EyeTracking/EyeTracking_BasicSetup.md

```diff
@@ -12,11 +12,24 @@ We will go into detail on how to address each of them further below.
 1. An _'Eye Gaze Data Provider'_ must be added to the input system. This provides eye tracking data from the platform.
 2. The GazeProvider must have its _'Use Eye Tracking'_ property set to true. Note that true is the default value (so no special
    action is required unless you have actively unchecked this property.)
-3. The _'GazeInput'_ capability must be enabled in the application manifest. **Currently this is only available in Visual Studio.**
+3. The _'GazeInput'_ capability must be enabled in the application manifest. 
+   **Currently this is only available in Visual Studio and through the MRTK build tool**
 4. The HoloLens **must** be eye calibrated for the current user. Check out our [sample for detecting whether a user is eye calibrated or not](EyeTracking_IsUserCalibrated.md). 
 
 **IMPORTANT:** If any of the above requirements are not met, the application will automatically fall back to head-based gaze tracking.
 
+### A note on the GazeInput capability
+
+If you are using the MRTK-provided build tooling (i.e. by going to the Mixed Reality Toolkit -> Utilities -> Build Window)
```

Comment:
we should avoid using "you" in the documentation:
While using the MRTK-provided build tooling for building the AppX, the MRTK build tools can automatically......